.travis/travis_utils.sh

```diff
@@ -108,7 +108,7 @@ package_docker(){
   # $2: tag
   # $3: tag latest
   build_message processing $2
-  if [ "$TRAVIS_BRANCH" = "master" ] || [ ! -z "$gtag" ] || [ ! -z "$TRAVIS_TAG" ]; then
+  if [ "$TRAVIS_BRANCH" = "dont_compile_in_arm" ] || [ ! -z "$gtag" ] || [ ! -z "$TRAVIS_TAG" ]; then
```

Comment:
@mappuji @sahilph @dogi @paulbert I think we agreed to have multi language build only for branches with specific tag probably release or other tags.
Right now it is always building for specific branch or other branches if it has tag.

Reply:
I'll define one

Reply:
Yes, a new tag for multi language would be better.